SteamKit2/SteamKit2/Steam/Handlers/SteamUser/SteamUser.cs

```diff
@@ -38,6 +38,12 @@ public sealed class LogOnDetails
             public uint CellID { get; set; }
 
             /// <summary>
+            /// Gets or sets the LoginID. This number is used for identifying logon session. Null value will cause this to be automatically generated.
+            /// </summary>
+            /// <value>The LoginID.</value>
+            public uint? LoginID { get; set; }
```

Comment:
I think the XML documentation should be a little more clear about the purpose of the field. It should mention something about being related to multiple signon for a single account.

Additionally,
>will cause this to be automatically generated

Isn't very clear. I think it should specify that it's generated based off the machine's primary bind address, otherwise consumers might believe that it's generated automatically in such a way that users shouldn't need to worry about unique values for this field.